app/src/main/java/com/example/playlistmaker/player/domain/PlayerInteractor.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.playlistmaker.player.domain
+
+import com.example.playlistmaker.player.domain.models.Track
+
+interface PlayerInteractor : Player {
```

Comment:
Здесь лучше использовать не наследование, а композицию) PlayerInteractorImpl может внутри себя использовать любой необходимый ему класс из слоя data (в твоем случае Player) для того, чтобы выполнять запросы, приходящие из активити. 

```suggestion
interface PlayerInteractor {
```